epm/pkg/epm-api/v1alpha1/api.proto

```diff
@@ -28,6 +28,8 @@ service EnclavePoolManager {
     rpc GetCache(GetCacheRequest) returns (GetCacheResponse) {}
     // SaveCache represents save the data to a cache directory and record the cache metadata
     rpc SaveCache(SaveCacheRequest) returns (SaveCacheResponse) {}
+    // SaveFinalCache represents save the enclave info which can be used instantly
+    rpc SaveFinalCache(SaveCacheRequest) returns (SaveCacheResponse) {}
```

Comment:
What is the difference between SaveCacheFinal and SaveCache, why not use the SaveCache directly?